features/interview/agents/nodes/question_generator.py

```
@@ -22,6 +22,24 @@
 
+def _invoke_with_retry(
+    chain,
+    prompt_variables: dict[str, object],
+    max_retries_per_question: int,
+) -> str:
+    """질문 생성 LLM 호출 재시도"""
+    last_error: Exception | None = None
+    for _ in range(max_retries_per_question + 1):
+        try:
+            response = chain.invoke(prompt_variables)
+            return response.content
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)